index.js

```diff
@@ -36,6 +36,15 @@ Command.prototype.historyJumpBackward = function historyJumpBackward() {
 	return false
 };
 
+
+Command.prototype.toggleShowHiddeFiles = function toggleShowHiddeFiles() {
+	var view = this.GUI.activeView().model;
+	view.showHiddenFiles = !view.showHiddenFiles;
+	view.updateGridViewData(false);
```

Comment:
To save state: Insert this code here: 
```javascript
applicationController.config.settings.showHiddenFiles = view.showHiddenFiles;
applicationController.config.save();
```

If we want to save state per view use:
`applicationController.config.settings.fileSystemViews[this.id]`

It is confusing and hard to get because `applicationController` is global :/. See here for API https://github.com/s-a/user-appdata#methods. I created an issue here https://github.com/sulu-one/sulu/issues/42